packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.tsx

```diff
@@ -103,6 +103,7 @@ export class HoverCard extends BaseComponent<IHoverCardProps, IHoverCardState> {
         { children }
         { isHoverCardVisible &&
           <ExpandingCard
+            key={ expandingCardProps!.expandedCardHeight! }
```

Comment:
this would be a very hard way to refresh the card. Basically it would reset its internal state and create a new one. A better approach might be to pass in height as a prop and let the card do what it wants with that.